api/src/services/export/feedback-mapper.js

```diff
@@ -1,11 +1,15 @@
 const moment = require('moment');
 const db = require('../../db');
 
-const formatDate = date => {
+const formatDate = (date, filters) => {
   if (!date) {
     return '';
   }
-  return moment(date).valueOf();
+  if (filters && filters.human && filters.human === 'true'){
```

Comment:
This check is quite cumbersome. In the export-data controller we try and clean this up so the service doesn't have to worry, for example, on api/src/controllers/export-data.js line 56 we have this:

```js
      if (filters.valid) {
        filters.valid = (filters.valid === 'true');
      }
```

That way the service and mappers only need to check a boolean and the request specific details are removed.